test/cucumber/features/steps/platformSteps.js

```diff
@@ -64,18 +70,21 @@ Then("The volume {string} should be created", async function (volumeName) {
   expect(volumes.Volumes).to.have.lengthOf(1);
 });
 
-Then("The service {string} should be removed", async function (serviceName) {
+Then('The service {string} should be removed', async function (serviceName) {
   const services = await docker.listServices({
     filters: { name: [`instant_${serviceName}`] },
   });
-  expect(services).to.be.array();
-  expect(services).to.be.empty;
+  const filterServicesWithExactName = services.filter((e) => e.Spec.Name === `instant_${serviceName}`);
```

Comment:
Refactor this into a function called "filterServicesWithExactName(services, serviceName )" since it's used many times ?

Reply:
done:)